services/essentials.yaml

```diff
@@ -39,27 +45,57 @@ spec:
     - groupId: org.jenkins-ci.plugins
       artifactId: junit
       version: 1.26.1
+    - groupId: org.jvnet.hudson.plugins
+      artifactId: favorite
+      version: 2.3.2
     - groupId: com.coravy.hudson.plugins.github
       artifactId: github
-      version: 1.29.2
+      version: 1.29.3
+    - groupId: org.jenkins-ci.plugins
+      artifactId: github-api
+      version: '1.92'
     - groupId: org.jenkins-ci.plugins
       artifactId: github-branch-source
-      version: 2.3.6
+      version: 2.4.0
     - groupId: org.jenkins-ci.plugins
       artifactId: htmlpublisher
-      version: '1.16'
+      version: '1.17'
+    - groupId: org.jenkins-ci.plugins
+      artifactId: jackson2-api
+      version: 2.9.7.0-beta
     - groupId: org.jenkins-ci.plugins
       artifactId: mailer
       version: '1.21'
+    - groupId: org.jenkins-ci.plugins
+      artifactId: matrix-project
+      version: '1.13'
     - groupId: org.jenkins-ci.plugins
       artifactId: mercurial
       version: '2.4'
     - groupId: org.jenkins-ci.plugins
       artifactId: ssh-agent
-      version: '1.16'
+      version: '1.17'
     - groupId: org.jenkins-ci.plugins.workflow
       artifactId: workflow-aggregator
       version: '2.6'
+    - groupId: org.jenkins-ci.plugins.workflow
+      artifactId: workflow-api
+      version: 2.30-beta-1
+    - groupId: org.jenkins-ci.plugins.workflow
+      artifactId: workflow-cps
+      version: 2.58-beta-1
+    - groupId: org.jenkins-ci.plugins.workflow
+      artifactId: workflow-cps-global-lib
+      version: '2.12'
+    - groupId: org.jenkins-ci.plugins.workflow
+      artifactId: workflow-job
+      version: 2.26-beta-1
+    - groupId: org.jenkins-ci.plugins.workflow
+      artifactId: workflow-scm-step
+      version: '2.7'
+    - groupId: org.jenkins-ci.plugins.workflow
+      artifactId: workflow-support
+      version: 2.21-beta-1
```

Comment:
This block is what I was actually trying to accomplish: pull in [JEP-210](https://jenkins.io/jep/210).